deploy/services-demo/demo.sh

```diff
@@ -108,6 +109,7 @@ run_haskell_service gundeck ${blue} Info
 run_haskell_service cannon ${orange} Info
 run_haskell_service cargohold ${purpleish} Info
 run_haskell_service proxy ${redish} Info
+run_haskell_service spar ${orange} Info
```

Comment:
Yes, double-orange is better than red, as in the integration tests.  I'm going to change that there, too.